core/test/unit/models/post_spec.js

```diff
@@ -1890,4 +1890,84 @@ describe('Unit: models/post', function () {
             });
         });
     });
+
+    describe('Mobiledoc conversion', function () {
+        let config = require('../../../server/config');
+        let origConfig = _.cloneDeep(config);
```

Comment:
You can use the configUtils to modify config

e.g. 

`configUtils = require('../utils/configUtils')`
`configUtils.set(enableDeveloperExperiments, true)`
`afterEach configUtils.restore()`